aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -153,12 +157,15 @@ class TORCH_CUDA_API FilterDescriptor
 public:
   void set(const at::Tensor &t, int64_t pad = 0, bool force_nhwc = false);
 
+  void print();
 private:
   void set(cudnnDataType_t dataType, int dim, int* size, cudnnTensorFormat_t filter_format) {
     AT_CUDNN_CHECK(cudnnSetFilterNdDescriptor(mut_desc(), dataType, filter_format, dim, size));
   }
 };
 
+std::ostream& operator<<(std::ostream & out, const FilterDescriptor& d);
```

Comment:
You might want to mark this with `TORCH_CUDA_API`

Reply:
`TORCH_CUDA_API` here introduces failed test, (not sure if they are relevant) like
```
Sep 25 19:10:06 Exception occurred:
Sep 25 19:10:06   File "/opt/conda/lib/python3.6/site-packages/sphinx/domains/cpp.py", line 6099, in _parse_type
Sep 25 19:10:06     raise self._make_multi_error(prevErrors, header)
Sep 25 19:10:06 sphinx.util.cfamily.DefinitionError: Error when parsing function declaration.
Sep 25 19:10:06 If the function has no return type:
Sep 25 19:10:06   Error in declarator or parameters-and-qualifiers
Sep 25 19:10:06   Invalid C++ declaration: Expecting "(" in parameters-and-qualifiers. [error at 15]
Sep 25 19:10:06     TORCH_CUDA_API std::ostream & operator<< (std::ostream &out, const FilterDescriptor &d)
Sep 25 19:10:06     ---------------^
Sep 25 19:10:06 If the function has a return type:
Sep 25 19:10:06   Error in declarator or parameters-and-qualifiers
Sep 25 19:10:06   If pointer to member declarator:
Sep 25 19:10:06     Invalid C++ declaration: Expected '::' in pointer to member (function). [error at 28]
Sep 25 19:10:06       TORCH_CUDA_API std::ostream & operator<< (std::ostream &out, const FilterDescriptor &d)
Sep 25 19:10:06       ----------------------------^
Sep 25 19:10:06   If declarator-id:
Sep 25 19:10:06     Invalid C++ declaration: Expecting "(" in parameters-and-qualifiers. [error at 28]
Sep 25 19:10:06       TORCH_CUDA_API std::ostream & operator<< (std::ostream &out, const FilterDescriptor &d)
Sep 25 19:10:06       ----------------------------^
```
https://dr.pytorch.org/api/view-log-full?build_id=137561284
https://dr.pytorch.org/api/view-log-full?build_id=137564136

The `operator<< TensorDescriptor` next to it doesn't have `TORCH_CUDA_API`, so I guess we don't need it?